### ISSUE TITLE
Picker not show up

### ISSUE THREAD

**Issue by Phantasimay** [fmbarina/pick-lsp-formatter.nvim#1]
I use your plugin on nvim nighty the formater are working but no UI show up. So I can't pick my desire formater

**Comment by fmbarina** [comment]
Hey, thanks for opening this issue.

Genuinely curious as to what could be causing it. If I'm reading this right, you used to get a picker, but no picker at all is appearing after a nightly update, even though the buffer still gets formatted. Can you confirm whether this is the case? Please clarify if not.

I'm a little inexperienced with this, so let's try taking it step by step. Hopefully, we'll identify the cause quickly, or at worst, gather information that'll help us do so later.

Remember to check with `:LspInfo` (assuming it's available) that there are actually multiple language servers attached to the buffer where the problem occurs. More details about your setup could help as well. From the top of my head, your precise Neovim version (`nvim --version`), plugin (pick-lsp-formatter) configuration, and which picker you used before this started happening.

I'll be waiting for your response.

**Comment by Phantasimay** [comment]
yes you are right. it dont appear at all. i was plant to use it with efm but after alot update of main plugin(treesitter, lspconf,lazy,mason) and neovim itself. all ecosystem around efm not working. i was tried to rollback any plugin and neovim itself but noting work. may be after big update some thing not compatible. now i give up with efm ecosystem. and rebuild neovim plugin set from other alternative plugin.

NB: 
i install your plugin after big update.

**Comment by fmbarina** [comment]
Well, I hope you're having better luck with plugins now. I'll be closing this, feel free to bump if you need anything.
